runtime/src/actor_error.rs

```diff
@@ -202,3 +204,12 @@ impl<T> AsActorError<T> for Option<T> {
         self.ok_or_else(|| ActorError { exit_code: code, msg: f().to_string() })
     }
 }
+
+pub fn extract_return<T>(ret: Option<IpldBlock>) -> Result<T, ActorError>
```

Comment:
nit: `deserialize_block`? I'm only suggesting it because this is _used_ for return values, but doesn't only apply to return values.

Reply:
Or `decode_block`? Not sure, it's just that calling this `extract_return` is a bit funky.

(but naming is hard anyways).